src/xpk/commands/cluster.py

```diff
@@ -500,12 +493,7 @@ def run_gke_cluster_create_command(
 
     if args.enable_pathways:
       enable_ip_alias = True
-      command += (
-          f' --create-subnetwork name={args.cluster}-subnetwork'
-          ' --cluster-dns=clouddns'
-          ' --cluster-dns-scope=vpc'
-          f' --cluster-dns-domain={args.cluster}-domain'
-      )
+      command += f' --create-subnetwork name={args.cluster}-subnetwork'
```

Comment:
I wonder if this is still needed? do you remember what the subnetwork is for again?

Reply:
The subnetwork was added to unblock Pathways workload from errors like "IP address exhausted in default network", etc. I will check separately, if it can be removed safely now and address that in another PR.

Reply:
Ah that type of problem we should solve generally in xpk, it also occurs in the normal non-pathways case too! Maybe a new xpk argument --create-network?

Reply:
Otherwise we get conflicts like https://buganizer.corp.google.com/issues/378732863 where we need a different network